src/command/QueueCommands.cxx

```diff
@@ -53,14 +53,15 @@ AddUri(Client &client, const char *uri, const Tag &tag)
 }
 
 static CommandResult
-AddDatabaseSelection(Client &client, const char *uri,
-		     gcc_unused Response &r)
+AddDatabaseSelection(Client &client, const RangeArg &range, const char *uri, gcc_unused Response &r)
```

Comment:
This whole commit is unnecessary because the MPD protocol can do this better already.

Reply:
Just for caryaudio.